db/schema.rb

```diff
@@ -52,13 +52,11 @@
   end
 
   create_table "manual_points", force: :cascade do |t|
-    t.bigint "member_id"
```

Comment:
These deletions are critically bad and that is due to the migration change that you made. We will need to do some merge conflicts even if the migration is fixed.